services/terraformer/templates/gcp.tpl

```diff
@@ -80,13 +80,48 @@ resource "google_compute_instance" "{{ $node.Name }}" {
   metadata = {
     ssh-keys = "root:${file("./public.pem")}"
   }
-  metadata_startup_script = "echo 'PermitRootLogin without-password' >> /etc/ssh/sshd_config && echo 'PubkeyAuthentication yes' >> /etc/ssh/sshd_config && service sshd restart"
+  metadata_startup_script = <<EOF
+# Allow ssh as root
+echo 'PermitRootLogin without-password' >> /etc/ssh/sshd_config && echo 'PubkeyAuthentication yes' >> /etc/ssh/sshd_config && service sshd restart
+
+{{- if not $nodepool.IsControl }}
+# Mount managed disk only when not mounted yet
+if ! grep -qs "/dev/sdb" /proc/mounts; then
```

Comment:
same comment as in hetzner, with one difference being *why is there not shebang here?*